DigitalLearningSolutions.Web/Scripts/trackingSystem/reports.ts

```diff
@@ -1,7 +1,15 @@
 import Chartist from 'chartist';
 import getPathForEndpoint from '../common';
 
-showMoreRows(false);
+const path = getPathForEndpoint('TrackingSystem/Centre/Reports/Data');
+const activityToggleRowClass = 'js-toggleable-activity-row';
+const activityToggleableRowDisplayNone = 'none';
+const activityToggleableRowDisplayTableRow = 'table-row';
+const request = new XMLHttpRequest();

```

Comment:
I think it's worth moving this down to just before `request.onload = ...`. The constants above this line are setting up data so I agree that they belong up here. But this is really part of the page-load action. In fact, if you extracted all of the code that deals with this request into a function and called that on page load, that would make the file a lot tidier.